compact.cpp

```diff
@@ -32,7 +32,7 @@ unsigned __stdcall ObjectMemory::lastOTEntry()
 	const OTE* pOT = m_pOT;
 	while (pOT[i].isFree())
 	{
-		ASSERT(i >= 0);
+		ASSERT(0 < i);
```

Comment:
Since 'i' is unsigned, it will always be greater than or equal to zero. If the value of 'i' is zero, then the decrement would wrap to the max integer value. This could be an endless loop. Again, almost certainly won't happen--I assume that some pOT[i] entry will return false from isFree(). I'm happy to revert it; it just was something in the list of issues.